Core/src/RestTrait.php

```diff
@@ -145,7 +145,7 @@ private function getApiEndpoint($default, array $config, string $apiEndpointTemp
             );
         }
 
-        if (!isset($config['universeDomain'])) {
+        if (!isset($config['universeDomain']) || empty($config['universeDomain'])) {
```

Comment:
This value is set by us, so we don't really need a check to make sure we don't supply a falsy value, only to make sure the function isn't invoked incorrectly 
```suggestion
        if (!isset($config['universeDomain'])) {
```